src/Zend/Form/Element/Captcha.php

```diff
@@ -64,6 +64,7 @@ public function getCaptcha()
      *
      * @param string|array|Zend_Captcha_Adapter $captcha
      * @param array $options
+     * @return $this
```

Comment:
Shouldn't this be returning a type?

Reply:
Yeah, `$this` is the type.  `$this` can be used when you want to say that it's a fluent interface.  A lot of times Zend will return the class name `Zend_Form_Element_Captcha` in this case, but that's not entirely correct, as it could mean that the return type is a different object with the same class.

The php typehint equivalent of `$this` would be `self`.  I think `self` can be used in place of `$this` in the docblock `@return` as well, but I've been using `$this` when it's a fluent interface.  Both PHPStan and Psalm seem happy with that.